feature_engine/transformation/yeojohnson.py

```diff
@@ -152,3 +152,45 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             X[feature] = stats.yeojohnson(X[feature], lmbda=self.lambda_dict_[feature])
 
         return X
+
+    def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Apply the inverse-transform function Yeo-Johnson.
+
+        Parameters
+        ----------
+        X: Pandas DataFrame of shape = [n_samples, n_features]
+            The data to be inversed.
+
+        Returns
+        -------
+        X: pandas dataframe
```

Comment:
Also the return, should be the same in all transformers. See this:

https://github.com/feature-engine/feature_engine/blob/3343305a01d1acfeff846b65d33a5686c6e8c84f/feature_engine/transformation/log.py#L173-L185

Could you update the docstrings here as well please?

Reply:
Change the entire dosctring to this

```python
        """
        Convert the data back to the original representation.

        Parameters
        ----------
        X: Pandas DataFrame of shape = [n_samples, n_features]
            The data to be transformed.

        Returns
        -------
        X_tr: pandas dataframe
            The dataframe with the transformed variables.
        """
```